tensorrec/tensorrec.py

```diff
@@ -87,6 +87,9 @@ def __init__(self, n_components=100,
             'tf_item_feature_values', 'tf_interaction_indices', 'tf_interaction_values', 'tf_learning_rate', 'tf_alpha',
             'tf_sample_indices', 'tf_n_sampled_items'
         ]
+        self.biased_graph_tensor_hook_attr_names = [
```

Comment:
Instead of adding a whole new list of graph hooks, we could get the same behavior by sticking these two new hooks in to the old list if `self.biased` is True.

Reply:
Oh right, that makes a lot more sense. Will do.